### ISSUE TITLE
...

### ISSUE THREAD

**Issue by populated** [bytevn/skidraider#1]
Please, NEVER, I mean NEVER, post code again. This is ass.

**Comment by 0xhorrored** [comment]
he cant code what do u except

**Comment by 0xhorrored** [comment]
at least my english is better than your code lmfao

**Comment by 0xhorrored** [comment]
yeah but you dont have one,howd you know that,also according to google ,learning english makes your code more readable, efficient, and easier to debug + ion wanna waste time arguing w retards,thanks
